### PR TITLE
fix: move `modelerTemplateIcon` to property

### PR DESCRIPTION
**Pull request by marstamm**
No change in behavior, even the tests are happy :smile: 
image

### CODE REVIEW

File: src/util/index.js [hunk 1]
```diff
@@ -1,31 +1,8 @@
 import {
-  getBusinessObject,
-  is
+  getBusinessObject
 } from 'bpmn-js/lib/util/ModelUtil';
 
 export function getModelerTemplateIcon(element) {
-  var modelerTemplateIcon = findExtension(element, 'zeebe:ModelerTemplateIcon');
-  return modelerTemplateIcon && (modelerTemplateIcon.$body || modelerTemplateIcon.body);
+  var modelerTemplateIcon = getBusinessObject(element).get('zeebe:modelerTemplateIcon');
```
Comment (pinussilvestrus) [inline src/util/index.js:6]: Uh nice, so this even works without `zeebe-bpmn-moddle` available (since the test is green)?

Comment (marstamm) [reply src/util/index.js:6]: Worked locally without the bump, but I bumped the moddle with [59ed83b](https://github.com/bpmn-io/element-templates-icons-renderer/pull/3/commits/59ed83b30a65086f7b92a1773cdf7ead122595ce)

Comment (pinussilvestrus) [reply src/util/index.js:6]: So I double-checked this locally with the [existing test case (no moddle descriptors)](https://github.com/bpmn-io/element-templates-icons-renderer/blob/main/test/spec/ElementTemplatesIconsRenderer.spec.js#L86).

image

✅

**Review by pinussilvestrus — APPROVED** [review]

File: src/util/index.js (hunk 1 above)
Comment (nikku) [reply src/util/index.js:6]: Only works without moddle if the prefix is `zeebe`, not some `xmlns:otherFooBar="https://zeebe-ns-url"`.

**Pull request merged**